database.rules.json

```diff
@@ -21,6 +21,10 @@
     "speakers": {
       ".read": true,
       ".write": "root.child('users').child(auth.uid).child('rules').child('speakers').val() === true"
+    },
+    "subscribers": {
+       ".read": false,
+       ".write": true
```

Comment:
how that would be possible? We shoul allow to write here, even if user isn't login